metricbeat/tests/system/metricbeat.py

```diff
@@ -65,7 +65,7 @@ def assert_no_logged_warnings(self, replace=None):
         """
         log = self.get_log()
 
-        pattern = self.build_log_regex("[cfgwarn]")
+        pattern = self.build_log_regex("\[cfgwarn\]")
```

Comment:
This was actually a bug in our code and removed all WARN and ERROR entries from the log :-(